### PR TITLE
Fix Credential Invalid Issuer Error

### PR DESCRIPTION
**Pull request by imhassantariq**
Fix the issue where Devstack gives the error of `invalid issuer` while Oauth

### CODE REVIEW

File: credentials/settings/devstack.py [hunk 1]
```diff
@@ -75,3 +75,8 @@
     'HELP_URL': os.environ.get('HELP_URL', 'http://edx.devstack.lms:18000/help'),
     'CONTACT_URL': os.environ.get('CONTACT_URL', 'http://edx.devstack.lms:18000/support/contact_us')
 }
+
+#####################################################################
```
Comment (danialmalik) [inline credentials/settings/devstack.py:79]: Can we do this in a separate PR ?

Comment (imhassantariq) [reply credentials/settings/devstack.py:79]: I am going to make 2 PR on upstream but here 1 PR is enough? What do you say?
